awx/ui/client/legacy/styles/codemirror.less

```diff
@@ -31,8 +31,8 @@
     background-color: @default-no-items-bord;
 }
 
-.CodeMirror-lineNumber {
-    color: @default-icon;
+.CodeMirror-linenumber {
+    color: @at-gray-70;
```

Comment:
![image](https://user-images.githubusercontent.com/214912/49091611-38ca4e80-f22e-11e8-95a1-de70c04d7f6b.png)

Reply:
`#707070` was the sweet spot here.